config/locales/nl.yml

```diff
@@ -220,7 +220,7 @@ nl:
   applications:
     invalid_url: De opgegeven URL is ongeldig
   auth:
-    agreement_html: Wanneer je op registeren klikt ga je akkoord met <a href="%{rules_path}">onze gebruikersvoorwaarden</a> en <a href="%{terms_path}">ons privacybeleid</a>.
```

Comment:
@ykzts I am a native Dutch speaker and I translated 99% of Mastodon. Actually I fixed my own typo. So hopefully you understand I find this very strange.